package vec.domain.exception

/**
 * セキュリティに関する例外です。
 * 認証、認可、その他理由で実行できないときに投げられます。
 */
class SecurityException : RuntimeException()
